src/workflows/modals/PermissionsModal.test.tsx

```diff
@@ -118,7 +118,7 @@ describe('PermissionsModal', () => {
     });
 
     // ASSERT
-    expect(screen.getByText('Edit permissions for namespace my-namespace'));
+    expect(screen.getByText('Edit permissions for collection my-namespace'));
```

Comment:
maybe change my-namespace to my-collection?

Reply:
`my-namespace` is what is being passed on L111. I didn't rename instances of namespace to collection here since this is a test file.